templates/puppetserver-ingress-masters.yaml

```diff
@@ -2,6 +2,9 @@
 {{- $releaseName := .Release.Name -}}
 {{- $serviceName := include "puppetserver.puppetserver-masters.serviceName" . }}
 {{- $servicePort := .Values.puppetserver.masters.service.ports.puppetserver.port -}}
+{{- $pathType := .Values.puppetserver.masters.ingress.pathType | default "ImplementationSpecific" -}}
+{{- $apiIsStable := eq (include "puppetserver.ingress.isStable" .) "true" -}}
+{{- $ingressSupportsPathType := eq (include "puppetserver.ingress.supportsPathType" .) "true" -}}
 apiVersion: networking.k8s.io/v1
```

Comment:
Shouldn't this use `puppetserver.ingress.apiVersion`?

Reply:
Not exactly - but tweaked it further. Thank you!